bokehjs/src/less/canvas_panel.less

```diff
@@ -1,6 +1,8 @@
 :host {
   position: relative;
   pointer-events: none;
+  align-items: start;
+  justify-items: start;
```

Comment:
Perhaps I should move these rules below into the blocks that declare `display: grid`. The problem with CSS grids is that they default to stretching items in both directions within a grid cell, i.e.:

```
align-items: stretch;
justify-items: stretch;
```

Which means that if we do not set the legend element's height to `max-content`, then the legend element will take up the entire height of the canvas panel (if it is rendered inside of a canvas panel).

But we need to remove the max-content height rule from the legend element because of a browser bug: [In WebKit, absolutely positioned grid with max-content, min-content or 100% height takes up the entire height of its nearest positioned ancestor](https://codepen.io/gabalafou-the-looper/pen/zxOKBxO).

Reply:
Should I move these rules into the same blocks below that declare `display: grid`?